app/javascript/packs/SignUpPage.js

```diff
@@ -10,14 +10,16 @@ if (isIE11) {
 }
 
 document.addEventListener('DOMContentLoaded', () => {
-  const signupPageContainer = document.getElementById('signup-page-container')
+  const loginPageContainer = document.getElementById('login-page-container')
   if (isIE11) {
-    signupPageContainer.classList.add('isIe11', 'pf-c-page')
+    loginPageContainer.classList.add('isIe11', 'pf-c-page')
   }
 
-  const loginLayout = document.querySelector('.login-layout')
-  loginLayout.removeChild(document.getElementById('old-signup-page-wrapper'))
+  let oldLoginWrapper = document.getElementById('old-login-page-wrapper')
+  if (oldLoginWrapper.parentNode) {
+    oldLoginWrapper.parentNode.removeChild(oldLoginWrapper)
```

Comment:
`oldLoginWrapper.remove()` ?

Reply:
Unfortunately, `remove()` is not supported in IE11

Reply:
But we are using the babel polyfill in the end, no?

Reply:
The problem is `'@babel/polyfill'` doesn't polyfills this specific feature. I've tried.
We should include another polyfill: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove#Polyfill
But I think is too much, I wouldn't include it
@josemigallas @didierofrivia

Reply:
Let's leave it like this then